readme.md

```diff
@@ -8,9 +8,53 @@ Here's what the front page looks like in [light mode](/screencapture-gzweb-light
 [dark mode](/screencapture-gzweb-dark-2021-03-22-20_24_03.png).
 
 ## Running the project locally
+The easiest and fastest way to run the project without cluttering your machine is by using docker containers. However you should be able to setup this project on any operating system that supports Django. We have instructions for Ubuntu based linux distributions and for Windows 10. Both can be found below.
 
-You should be able to setup this project on any operating system that supports Django. We have instructions for Ubuntu based linux distributions and for Windows 10. Both can be found below.
-Alternatively you can run this project in containers, by using docker-compose.
+### 1. Install Docker
+
+##### Linux/Ubuntu
+```sh
+# Install Docker
+sudo apt-get update
+sudo apt-get install -y docker.io
+
+# Configure docker to start on boot
+sudo systemctl enable docker.service
+
+# Manage Docker as a non-root user
+sudo groupadd docker
+sudo usermod -aG docker $USER
+```
+Log out of your session completely and then log back in
+
+```sh
+# Install docker-compose
+sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+sudo chmod +x /usr/local/bin/docker-compose
+```
+```sh
+# Install command completion
+sudo curl \
+    -L https://raw.githubusercontent.com/docker/compose/1.29.2/contrib/completion/bash/docker-compose \
+    -o /etc/bash_completion.d/docker-compose
+source ~/.bashrc
+```
+##### Windows 10
+
+1. Follow the instructions in the Docker documentation https://docs.docker.com/desktop/windows/install/; the installation varies depending on the Windows 10 edition. 
```

Comment:
The "Docker documentation" bit should be 

`[Docker documentation](https://docs.docker.com/desktop/windows/install)`

The use of 

`Docker documentation https://docs.docker.com/desktop/windows/install/`

means that we provide a raw URL.

Reply:
Done!